Mathlib/Topology/Algebra/ContinuousMonoidHom.lean

```diff
@@ -413,3 +415,278 @@ theorem locallyCompactSpace_of_hasBasis (V : ℕ → Set Y)
 end LocallyCompact
 
 end ContinuousMonoidHom
+
+end
+
+section
+
+/-!
+
+# Continuous MulEquiv
+
+This section defines the space of continuous isomorphisms between two topological groups.
+
+## Main definitions
+
+-/
+
+universe u v
+
+variable (G : Type u) [TopologicalSpace G] (H : Type v) [TopologicalSpace H]
+
+/-- Define the structure of two-sided continuous isomorphism of additive groups. -/
```

Comment:
```suggestion
/-- The structure of two-sided continuous isomorphisms between additive groups. Note that both the map and its inverse have to be continuous. -/
```
I think we don't need to say "Define" -- remember that this is what people will read when they hover over the definition. Same comment for `ContinuousMulEquiv`.

Reply:
Thank you for the suggestion!